test/framework/cluster_proxy.go

```diff
@@ -253,6 +259,27 @@ func (p *clusterProxy) Apply(ctx context.Context, resources []byte, args ...stri
 	return exec.KubectlApply(ctx, p.kubeconfigPath, resources, args...)
 }
 
+// Create creates using the controller-runtime client.
+func (p *clusterProxy) Create(ctx context.Context, resources []byte) error {
```

Comment:
Why do we need a create func? Why not use GetClient().Create ?

I'm a bit concerned about silently ignoring IsAlreadyExists errors on this level (could lead to all sorts of surprises for the caller of this func if its YAML is not deployed)